packages/compass-query-bar/src/components/query-option.tsx

```diff
@@ -132,6 +138,38 @@ const QueryOption: React.FunctionComponent<QueryOptionProps> = ({
     [name, onChange]
   );
 
+  const onFocusCallback = useCallback(
```

Comment:
Is this a leftover?

Reply:
Yes, it was, I was just keeping it there while I was moving the code around.